plugins/background-remover/framer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "id": "ot052l",
+    "name": "Background Remover",
+    "modes": ["editImage"],
```

Comment:
I think it could be nice to support this on `canvas` mode as well and apply it to the selection if a layer with an image is selected 🙏

Reply:
Agreed! Canvas mode has now been added, multi-image select is also supported.